### PR TITLE
Bug 1149994 - added basic history UITest

### PR DESCRIPTION
**Pull request by AaronMT**
very basic, I can add more later as more functionality lands

### CODE REVIEW

**Comment by wesj** [comment]
This looks pretty good Aaron. Can you file a bug and add it as an attachment?

**Comment by AaronMT** [comment]
https://bugzilla.mozilla.org/show_bug.cgi?id=1149994

File: UITests/HistoryTests.swift [hunk 1]
```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```
Comment (wesj) [inline UITests/HistoryTests.swift]: If you want, bnicholson will cry if you don't fix the spacing here.

File: UITests/HistoryTests.swift [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+class HistoryTests: KIFTestCase {
+    private var webRoot: String!
+
+    override func setUp() {
+        webRoot = SimplePageServer.start()
+    }
+
+    /**
+    * Tests for listed history visits
+    */
```
Comment (wesj) [inline UITests/HistoryTests.swift]: Same here. Line up the *'s

**Comment by AaronMT** [comment]
I'm going to close and re-open a new, I'm bad at git.

**Pull request closed without merging**